src/System.Runtime/tests/System/FormattableStringTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Globalization;
+using Xunit;
+
+namespace System.Tests
+{
+    public class FormattableStringTests : RemoteExecutorTestBase
+    {
+        [Fact]
+        public static void CurrentCulture_ImplicityAndExplicitMethodsReturnSameString()
+        {
+            double d = 123.456;
+            string text1 = $"This will be formatted using current culture {d}";
+            string text2 = FormattableString.CurrentCulture($"This will be formatted using current culture {d}");
+            Assert.Equal(text1, text2);
+        }
+
+        [Fact]
+        public static void CurrentCulture_Null_ThrowsArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() => FormattableString.CurrentCulture(null));
+        }
+
+        [Fact]
+        public static void CurrentCulture_DutchCulture_ThrowsArgumentNullException()
+        {
+            RemoteInvoke(
+                () =>
+                {
+                    CultureInfo.CurrentCulture = new CultureInfo("nl");
+                    double d = 123.456;
+                    string text = FormattableString.CurrentCulture($"Dutch decimal separator is comma {d}");
+                    Assert.Equal("Dutch decimal separator is comma 123,456", text);
```

Comment:
I'm fine with the string being hardcoded here but you could have also used string.Format or something similar with an explicit CultureInfo.